recipes/datacube-compute/recipe.yaml

```diff
@@ -0,0 +1,53 @@
+context:
+  name: datacube-compute
+  version: 0.0.6
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/auspatious/datacube-compute/archive/${{ version }}.tar.gz
+  sha256: 4706e086e4fb5049c88cfd28ea2dfda2f40c56146834e09fb59e8956d5be801a
+
+build:
+  number: 0
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }} ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
For windows support
```suggestion
    - set SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }}  # [win]
    - export SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }}  # [unix]
    - ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
Thanks, that kinda worked, but only because both the `set` and `export` lines get ran, and the error doesn't block :sweat_smile: :

```
(base) %SRC_DIR%>set SETUPTOOLS_SCM_PRETEND_VERSION=0.0.6 
(base) %SRC_DIR%>export SETUPTOOLS_SCM_PRETEND_VERSION=0.0.6 
'export' is not recognized as an internal or external command,
operable program or batch file.
```

I think I've figured out the if-then statement with [CEP-14](https://github.com/conda/ceps/blob/main/cep-0014.md#script-section), something like this:

```
  script:
    - if: win
      then: set SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }}
    - if: unix
      then: export SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }}
```

Applied this at commit ff3310879a6fc146a677b633c608f4922f7bd54c